### ISSUE TITLE
Question: Check if back stack is empty

### ISSUE THREAD

**Issue by emartynov** [terrakok/Cicerone#138]
I can not find an easy way to get from the router if there is something in the back stack to pop.

**Comment by terrakok** [comment]
Yes, it's true cause router doesn't have direct access to navigator which works with fragment manager.  
Navigator lives in UI lifecycle and can be destroyed any time.  
Router is independent from UI lifecycle.  

But if it necessary I can add `Router.getCurrentStackSize(): Int?` method which returns `null` when navigator does not available. What do you think?

**Comment by aziko19** [comment]
when are you going to add this @terrakok . It's been 2 years already.
